packages/workshop-utils/src/epic-api.server.ts

```diff
@@ -70,10 +71,20 @@ export async function getEpicVideoInfos(
 	epicWebUrls?: Array<string> | null,
 	{ request, timings }: { request?: Request; timings?: Timings } = {},
 ) {
-	if (!epicWebUrls) return {}
+	const log = logger('epic-api:video-infos')
+	
+	if (!epicWebUrls) {
+		log('no epic web URLs provided, returning empty object')
+		return {}
+	}
+	
 	const authInfo = await getAuthInfo()
-	if (getEnv().EPICSHOP_DEPLOYED) return {}
+	if (getEnv().EPICSHOP_DEPLOYED) {
+		log('deployed mode, skipping epic video info fetch')
+		return {}
+	}
```

Comment:
This change is unnecessary. I won't need to debug videos in a deployed environment because I'll know that this feature doesn't work in that environment anyway.